### ISSUE TITLE
Can't set child item draggable range in children group

### ISSUE THREAD

**Issue by h3clikejava** [h6ah4i/android-advancedrecyclerview#13]
onGetChildItemDraggableRange() method only set child drag range in group session.But can't set range in itself session.Because I have a child item at last index, i need it can't draggable.
I can override onMoveChildItem() method when drag in a session.but when i drag it cross other session group,It's will crash.It's always move it.....Help me

**Comment by h6ah4i** [comment]
Hi. I've just pushed the update to the `develop` branch (https://github.com/h6ah4i/android-advancedrecyclerview/commit/7c48e0caa366339e1e191533c4cb1a8eead940a0).
The new `ChildPositionItemDraggableRange` class will helps you :) Please refer to the updated example code.

**Comment by h6ah4i** [comment]
Fixed in 0.7.0.

Introduced these new classes:

- `ChildPositionItemDraggableRange`
- `GroupPositionItemDraggableRange`


Usage:

```
    @Override
    public ItemDraggableRange onGetGroupItemDraggableRange(MyGroupViewHolder holder, int groupPosition) {
        final int start = findFirstSectionItem(groupPosition);
        final int end = findLastSectionItem(groupPosition);

        return new GroupPositionItemDraggableRange(start, end);
    }

    public ItemDraggableRange onGetChildItemDraggableRange(MyChildViewHolder holder, int groupPosition, int childPosition) {
        final int start = 0;
        final int end = 2;
        return new ChildPositionItemDraggableRange(start, end);
    }
```
